import java.io.*;
class TryCatch6 {
    public static void main(String args[]) {
        int a = 5, b = 0, c;
        int arr[] = new int[5];
        String str = null;
        try {
            c = a / b;              
            arr[10] = 200;           
            System.out.println("The length of the String is " + str.length());  
        }
        catch (NullPointerException NP) {
            System.out.println("NullPointerException caught");
        }
        catch (ArithmeticException ae) {
            System.out.println("ArithmeticException caught");
        }
        catch (ArrayIndexOutOfBoundsException ai) {
            System.out.println("ArrayIndexOutOfBoundsException caught");
        }
        catch (Exception e) {
            System.out.println("Exception caught");
        }
    }
}


